docs/development.md

```diff
@@ -22,6 +22,16 @@ just prepare
 just compile-commands
 ```
 
+## Dependencies
+
+In order to install dependencies, please run the following command:
```

Comment:
Nit: Maybe add "Also refer to Building workerd in README.md for the list of required dependencies by platform"